### ISSUE TITLE
How to add custom headers?

### ISSUE THREAD

**Issue by bigperson** [dai-siki/vue-image-crop-upload#4]
Hi, i use jwt auth.
How to add custom headers?

method in `:other-params="otherParams"` or method in `:params="otherParams"` : 
```
otherParams: {
    headers: {
        'Authorization': 'Bearer ...'
   }
}
```
not working :-(

vue 2.0.6

**Comment by bigperson** [comment]
My fix for upload-2.vue component:
add to props: 
```
 //Add custom headers
        headers: {
            type: Object,
            'default': null
        },
```
for method `upload()` add in:
```
let that = this,
                { ..., headers, ...}  = this,
                fmData = new FormData();

```
after `client.open('POST', url, true);`  add:
```
                for (let headerKey in headers) {
                    if (headers.hasOwnProperty(headerKey)) {
                        client.setRequestHeader(headerKey, headers[headerKey]);
                    }
                }
```

Example:
```
<image-crop-upload
                @crop-success="cropSuccess"
                @crop-upload-success="cropUploadSuccess"
                @crop-upload-fail="cropUploadFail"
                field="avatar"
                key="0"
                lang-type="en"
                v-model="show"
                :headers="headers"
                :url="getUploadUrl()"
        >
        </image-crop-upload>
```
```
headers: {
                  'Authorization': ....
              }
```
Please create pull request

**Comment by dai-siki** [comment]
Good job! This has been fixed in @ 1.3.1，Thank you very much for your contribution~
